src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -83,7 +87,10 @@ export class ExamChecklistComponent implements OnInit {
     checkPointsExercisesEqual() {
         this.pointsExercisesEqual = true;
         this.exam.exerciseGroups!.forEach((exerciseGroup) => {
-            const maxPoints = exerciseGroup.exercises?.[0].maxPoints;
+            let maxPoints: number | undefined = 0;
+            if (exerciseGroup.exercises && exerciseGroup.exercises!.length !== 0) {
+                maxPoints = exerciseGroup.exercises?.[0].maxPoints;
```

Comment:
The type here looks a bit odd. Personally I'd change this to
```suggestion
            let maxPoints = 0;
            if (exerciseGroup.exercises && exerciseGroup.exercises!.length !== 0) {
                maxPoints = exerciseGroup.exercises?.[0].maxPoints!;
```
(since we use `maxPoints!` already through the file.)